labs/lab18-helm-charts/README.md

```diff
@@ -0,0 +1,435 @@
+
+```
+Successfully packaged chart and saved it to: /Users/fernandocremer/test/tests/charts/bitnami/wordpress/wordpress-14.0.7.tgz
+
+```
+
+We will include this file in our Replicated Application Release.
+
+## Adding the Helm Chart to a Release
+
+You should have received an invite to log into https://vendor.replicated.com -- you'll want to accept this invite and set your password. 
+
+In this lab, we have already created an application for you, but there are no releases created yet. We are going to cover creating the release with the UI and through the CLI. Going through the UI is easier and better if you just want to understand the basic concepts as it will automatically create resources for you. With the CLI, we'll take a more 'starting from scratch' approach. Later in the lab we'll be making changes to the application, and will only cover the CLI.
+
+### Using the UI
+
+* Log in to the Vendor Portal using your email and password.
+* On the navigation menu, select **Releases**
+
+    <p align="center"><img src="img/releases.png" alt="vendor-ui" width="1000" margin=auto/></img></p>
+
+* Click on **Create Release**
+
+    <p align="center"><img src="img/create-release.png" alt="create-release" width="1000" margin=auto/></img></p>
+
+* We will not use the default nginx app, so let's delete the following files:
+    * example-configmap.yaml
+    * example-deployment.yaml
+    * example-service.yaml
+
+    <p align="center"><img src="img/delete-files.png" alt="files-to-delete" width="1000" margin=auto/></img></p>
+
+* Now we are ready to add the Chart! Open a window and browse to the directory where the Helm Chart we created is located. From your terminal on macos, you can use `open .` to quickly get a Finder window open.
+* Drag and Drop it where the Replicated Release Files navigator as shown below
+    <p align="center"><img src="img/dragchart.png" alt="drag and drop chart" width="1000" margin=auto/></img></p>
+
+* You will be prompted to select which method to use when deploying the Helm Chart:
+   
+    <p align="center"><img src="img/helm-opts.png" alt="helm options" width="600" margin=auto/></img></p>
+
+   * **Native Helm** - This should be the default option and should be the one selected. With this option, App Manager will use Helm to deploy the Helm Chart in the customer's environment.
+   * **Replicated** - This option will eventually be depecrated and should not be selected. With this option, App Manager will first generate Kubernetes Manifests from the Helm Chart (i.e., run `helm template`) before they are deployed to the cluster as manifests.
+
+* Note how a new file is automatically created by Replicated that includes details about the chart we just added.
+
+<p align="center"><img src="img/helm-file.png" alt="helm file" width="600" margin=auto/></img></p>
+
+As a last step, save the release we've created 
+<p align="center"><img src="img/save release.png" alt="helm file" width="600" margin=auto/></img></p>
+
```

Comment:
not a blocker but just wanted to note this linter warning

<img width="688" alt="Screen Shot 2022-07-09 at 1 12 36 PM" src="https://user-images.githubusercontent.com/3730605/178118014-cb2c5161-3723-4a48-96b6-5e31d5a38f9d.png">

Reply:
I think this warning is expected. Even if you would point it to a valid resource as part of the Helm Chart. Only way to get rid of the warning would be not to use `statusInformers`. But that is also not really what we want.